### PR TITLE
Add meeting status notifier to React component lib meeting demo

### PR DESCRIPTION
**Pull request by devalevenkatesh**
This will help in testing various meeting statuses from Amazon Chime SDK component library for React.

**Issue #:**

**Description of changes:**
- Add a new component `MeetingStatusNotifier` to throw UI notifications based on meeting status updates from underlying Amazon Chime SDK component library for React.
- Update `amazon-chime-sdk-component-library-react` submodule which includes the new `MeetingStatus.Reconnecting` status.

**Testing**

1. How did you test these changes?

    1. Run the demo app.
    2. Verify correct connecting and connected statuses are shown in meeting screen.
    3. Turn internet off and verify reconnecting status is notified.
    4. Keep the internet off and see reconnecting statues comes up until the internet is connected back.
    5. Once internet is connected back verify meeting connected status is shown.
    6. Do the same again, but, this time do not turn ON the internet and verify meeting failed message is notified and you are re-directed to home page.

3. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
7. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ build
 apps/chat/src/Config.js
 apps/chat/src/backend/serverless/appconfig.json
 apps/chat/src/backend/serverless/.aws-sam
+ada*
```
Comment (ltrung) [inline .gitignore:9]: what is this :p Is this internal stuff :p

File: apps/meeting/CONTRIBUTING.md [hunk 2]
```diff
@@ -21,44 +21,7 @@ reported the issue. Please try to include as much information as you can. Detail
...
-
-3. Modify source and ensure local demo app testing works as expected.
-4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request interface.
-6. The PR should open against the *meeting-dev* branch and not the *main* branch.
-> Note that *meeting-dev* branch is for testing the next release, hence contributor must create PR against the meeting-dev branch.
-The *main* branch is a stable demo branch for the last release and any changes after a release must go into the *meeting-dev* branch.
-
-7. When opening PR, exclude any `package.json` changes for `the amazon-chime-sdk-component-library-react` dependency.
+Contributions via pull requests are much appreciated.
```
Comment (ltrung) [inline apps/meeting/CONTRIBUTING.md:24]: Do we need legal sign off on this :p

File: .gitignore (hunk 1 above)
Comment (devalevenkatesh) [reply .gitignore:9]: Yeah when you run the script (internal) to get credentials locally a log file is added, ignored that.

File: apps/meeting/CONTRIBUTING.md (hunk 2 above)
Comment (devalevenkatesh) [reply apps/meeting/CONTRIBUTING.md]: Done updated, also removed a stale section in README around contributing.

**Review by ltrung — APPROVED** [review]

**Pull request merged**